docs/ugcstor-csi.md

```diff
@@ -0,0 +1,487 @@
+---
+id: ugcstor-csi
+title: cStor User Guide
+sidebar_label: cStor
+---
+------
+<br>
+
+<img src="/docs/assets/svg/3-config-sequence.svg" alt="OpenEBS configuration flow" style="width:100%">
+
+<br>
+<br>
+
+This user guide section provides the operations needed to be performed by the User and the Admin for configuring cStor related tasks. 
+
+   :::note 
+   The recommended approach to provision cStor Pools is to use cStorPoolCluster(CSPC), the detailed steps have been provided in this document. However, OpenEBS also supports provisioning of cStor Pools using StoragePoolClaim (SPC). For detailed instructions, refer to the <a href="https://docs.openebs.io/v260/docs/next/ugcstor.html" target="_blank">cStor User guide(SPC)</a>.<br>
+   :::
+
+## User operations
+
+
+## Admin operations
+
+[Expanding a cStor volume](#expanding-a-cstor-volume)
+
+[Performance Tunings in cStor Pools](#performance-tunings-in-cstor-pools)
+
+<h2><a class="anchor" aria-hidden="true" id="user-operations"></a>User Operations</h2>
+
+<h2><a class="anchor" aria-hidden="true" id="user-operations"></a>Admin operations</h2>
+
+### <a class="anchor" aria-hidden="true" id="expanding-a-cstor-volume"></a>Expanding a cStor volume
+
+OpenEBS cStor introduces support for expanding an iSCSI PV using the CSI provisioner. Provided cStor is configured to function as a CSI provisioner, you can expand iSCSI PVs that have been created by cStor CSI Driver. This feature is supported with Kubernetes versions 1.16 and above.
```

Comment:
```suggestion
OpenEBS cStor introduces support for expanding an iSCSI PV using the CSI provisioner. Provided cStor is configured to function as a CSI provisioner, you can expand iSCSI PVs that have been created by cStor CSI Driver. This feature is supported with Kubernetes versions 1.19 and above.
```
CSIDriver is only available in storage.k8s.io/v1 since k8s v1.19. @shubham14bajpai please check here once.